comptests/prov/test_prov_data_only_deploy.py

```diff
@@ -0,0 +1,150 @@
+        LOGGER.info("Test Started.")
+        LOGGER.info("Step 1: Clone cortx-k8s script repo on setup.")
+        cmd = "cd {}; {}".format(cons.HA_TMP,
+                                 commands.CMD_GIT_CLONE.
+                                 format(self.prov_deploy_cfg["git_prov_k8_repo"].format(
+                                     self.script_remote_branch
+                                 )))
+        for node_obj in self.host_list:
+            node_obj.execute_cmd(cmd)
+        LOGGER.info("Step 1: Done.")
+
+        LOGGER.info("Step 2: Update solution.yaml with default data only deploy config.")
+        remote_sol_path = self.prov_deploy_cfg["git_remote_path"] + "solution.example.yaml"
+        solution_path = self.master_node_obj.copy_file_to_local(remote_path=remote_sol_path,
+                                                                local_path=self.local_sol_path)
+        assert_utils.assert_true(solution_path[0], solution_path[1])
+        resp = self.deploy_lc_obj.update_sol_for_granular_deploy(file_path=self.local_sol_path,
+                                                                 host_list=self.host_list,
+                                                                 image=self.cortx_data_image,
+                                                                 deployment_type=self.deploy_cfg[
+                                                                     "deployment_type_data"
+                                                                 ])
+        assert_utils.assert_true(resp[0])
+        for node_obj in self.host_list:
+            print(resp[1])
+            print(self.prov_deploy_cfg["git_remote_path"] + self.deploy_cfg["new_file_path"])
+            res = node_obj.copy_file_to_remote(local_path=resp[1],
+                                               remote_path=self.prov_deploy_cfg[
+                                                               "git_remote_path"
+                                                           ] + self.deploy_cfg["new_file_path"])
+            assert_utils.assert_true(res[0], res[1])
+        LOGGER.info("Step 2: Done.")
+
+        LOGGER.info("Step 3: Run prereq script on setup nodes.")
+        for node_obj in self.host_list:
+            self.deploy_lc_obj.execute_prereq_cortx(node_obj,
+                                                    self.prov_deploy_cfg["git_remote_path"],
+                                                    "/dev/sdb")
+        LOGGER.info("Step 3: Done.")
+
+        LOGGER.info("Step 4: Start data only cortx cluster.")
+        resp = self.deploy_lc_obj.deploy_cluster(self.master_node_obj,
+                                                 self.prov_deploy_cfg["git_remote_path"])
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 4: Done.")
+        LOGGER.info("Test Completed.")
+
+    @pytest.mark.lc
+    @pytest.mark.comp_prov
+    @pytest.mark.tags("TEST-39370")
+    def test_39370(self):
+        """
```

Comment:
what if this test runs first and there is no deployment, either you ca combine this test with above one.

Reply:
We want this to be different TC.